### PR TITLE
WIP Chapter 02 Rails

### PR DESCRIPTION
**Pull request by Xenosb**
This is unfortunately still in progress as it is taking me ages to write the materials for some reason. Will continue tomorrow.

### CODE REVIEW

File: chapters/02-ruby_on_rails/readme.md [hunk 1]
```diff
@@ -0,0 +1,795 @@
...
+charge of showing your page or API result to the users.
+
+## Ruby and gems
+
+If you have some experience with Python, PHP or Node, you might know of pip,
+composer or npm. All of these tools are package managers for their respective
+programming languages. They help you get all the tools that you need without
+copying source code to your project. Package manager for Ruby is called **gem**.
+
+```bash
```
Comment (Narayanan170) [inline chapters/02-ruby_on_rails/readme.md]: If they installed via rbenv, they already have the gem command.

File: chapters/02-ruby_on_rails/readme.md [hunk 2]
```diff
@@ -0,0 +1,795 @@
...
+application works. But as you can see this web doesn't really look that nice.
+Maybe we could fix that a bit?
+
+In this chapter we will use a few of these front-end technologies but they will
+be more thoroughly explained in chapter 5. So use resources and guides regarding
+css and javascript as is.
+
+[Bootstrap](http://getbootstrap.com) is an easy to use framework which will help
+us make pretty web pages without much effort! To install it we can use gem. Open
+`Gemfile` in the root of your project and add `gem 'bootstrap'` to include it to
```
Comment (Narayanan170) [inline chapters/02-ruby_on_rails/readme.md]: If I'm not mistaken, you need **bootstrap-sass** gem, not just bootstrap

Comment (Xenosb) [reply chapters/02-ruby_on_rails/readme.md]: bootstrap-sass is for bs 3 while bootstrap is used for v4

**Comment by Xenosb** [comment]
What do you mean with this small fixes?

File: chapters/02-ruby_on_rails/readme.md (hunk 1 above)
Comment (Xenosb) [reply chapters/02-ruby_on_rails/readme.md]: Will add now with an additional assignment.

**Comment by Narayanan170** [comment]
But can you use sass with the new gem?

**Comment by Narayanan170** [comment]
Didn't have any imagination xD

**Comment by Xenosb** [comment]
Yes, you can use sass on both versions of gem. "Only" change is bootstrap version. Take a look here https://github.com/Xenosb/ruby-homework

**Pull request merged**